utils/biospecimen.js

```diff
@@ -221,7 +221,8 @@ const biospecimenAPIs = async (req, res) => {
         }
         else {
             const { searchShipments } = require('./firestore');
-            const response = await searchShipments(siteCode);
+            const requestedSite = convertSiteLoginToNumber(req?.query?.requestedSite);
+            const response = requestedSite ? await searchShipments(requestedSite) : await searchShipments(siteCode);
```

Comment:
Are these changes needed here? Will this enable cross-site search?

Reply:
For now this will retrieve the corresponding biospecimen documents based on the login site of the [box's button clicked](https://github.com/episphere/biospecimen/pull/527/files#diff-3b1b3cdfa6b45cd7638f994ae8322d8e2b4e0e5b0b3732e282e7489a7711d5ceR212). It could be used for search functionality later on in BPTL dashboard.